backend/src/main/java/com/bakdata/conquery/models/query/concept/specific/CQAnd.java

```diff
@@ -74,10 +79,14 @@ public void collectResultInfos(ResultInfoCollector collector) {
 		}
 
 		if(createExists){
-			collector.add(new SimpleResultInfo(Objects.requireNonNullElse(getLabel(), "AND"), ResultType.BooleanT.INSTANCE));
+			collector.add(new SimpleResultInfo(Objects.requireNonNullElse(getLabel(), QueryUtils.createDefaultMultiLabel(children, " " + getGetC10nName() + " ")), ResultType.BooleanT.INSTANCE));
 		}
 	}
 
+	private static String getGetC10nName() {
+		return C10N.get(CQElementC10n.class, Locale.ROOT).and();
```

Comment:
Hier nicht die Root Locale nehmen sondern die, die in der I18n Klasse gesetzt ist

Reply:
done